doc/api/https.md

```diff
@@ -288,6 +289,10 @@ The following additional `options` from [`tls.connect()`][] are also accepted:
 string, it is automatically parsed with [`new URL()`][]. If it is a [`URL`][]
 object, it will be automatically converted to an ordinary `options` object.
 
+`https.request()` returns an instance of the [`http.ClientRequest`][]
+class. The `ClientRequest` instance is a writable stream. If one needs to
+upload a file with a POST request, then write to the `ClientRequest` object.
+
```

Comment:
Do we need any of this information in the text if it's been added to line 278 above? Maybe the paragraph can go but the other addition can stay?

```suggestion
```

Reply:
This paragraph was copied from http.request, includes the use of 'one'. All I did was add a single letter 's' to 'http': 

https://github.com/nodejs/node/blob/master/doc/api/http.md#httprequesturl-options-callback